src/rapidata/rapidata_client/workflow/evaluation_workflow.py

```diff
@@ -5,11 +5,12 @@
 class EvaluationWorkflow(Workflow):
     """Workflow to run evaluation orders. This is used internally only and should not be necessary to be used by clients."""
 
-    def __init__(self, validation_set_id: str):
+    def __init__(self, validation_set_id: str, should_accept_incorrect: bool):
         super().__init__("EvaluationWorkflow")
         self.validation_set_id = validation_set_id
+        self.should_accept_incorrect = should_accept_incorrect
```

Comment:
please add a docstring explaining what the parameters do. in the same style as the other classes

Reply:
I just added it.